dask-gateway-server/dask_gateway_server/backends/base.py

```diff
@@ -281,18 +281,25 @@ class ClusterConfig(Configurable):
 
     # Number of threads per worker. Defaults to the number of cores
     worker_threads = Integer(
-        min=1,
         help="""
         Number of threads available for a dask worker.
 
         Defaults to ``worker_cores``.
         """,
+        min=1,
         config=True,
+        allow_none=True,
     )
 
     @default("worker_threads")
     def _default_worker_threads(self):
-        return self.worker_cores
+        return max(int(self.worker_cores), 1)
+
+    @validate("worker_threads")
+    def _validate_worker_threads(self, proposal):
```

Comment:
I promise this was the first thing I tried before the messy new trait stuff I added :) But I think I only discovered the `int` rounding toward 0 issue after I gave up on this approach. Glad to see it's working now.